packages/framework/src/Resources/scripts/admin/plugins/jquery.counterup.js

```diff
@@ -51,6 +53,13 @@
                     }
                 }
 
+                // Preserve spaces if input had spaces
+                if (isSpace) {
+                    while (/(\d+)(\d{3})/.test(newNum.toString())) {
```

Comment:
we can use [0-9] for consistency, but i like `\d` more.
i dont think this will work for higher numbers like `23 000 000`

Reply:
This works, tested on milions